app/src/org/commcare/android/database/DbUtil.java

```diff
@@ -95,9 +94,6 @@ public static PrototypeFactory getPrototypeFactory(Context c) {
                         if (!emptyc) {
                             continue;
                         }
-                        if (Externalizable.class.isAssignableFrom(prototype)) {
-                            classNames.add(cn);
-                        }
```

Comment:
Removed this because the class name is added regardless right below. Sound alright?